presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -27,9 +27,16 @@ Linux Operating System
 Java Runtime Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-* 64-bit required
-* version 11.0.7+ (version 12+ not supported)
-* Azul Zulu recommended (most tested)
+Presto requires a 64-bit version of Java 11, with a minimum required version of 11.0.7.
+Newer patch versions such as 11.0.8 or 11.0.9 are supported. Earlier patch versions
+such as 11.0.2 will not work, nor will earlier major versions such as 8, 9, or 10.
+Newer major versions such as 12 or 13 are also not supported -- they may work,
```

Comment:
remove "also"

Reply:
Good point. It's not "also" since the previous one is "do not work" vs "not supported"